src/pages/profile.js

```diff
@@ -37,10 +61,25 @@ const Profile = ({ data, gitHubUsername }) => {
                 <a
                   className="github-link"
                   href={`https://www.github.com/${user.nickname}`}
+                  target="_blank"
+                  rel="noreferrer"
                 >
                   <GitHubIcon className="profile-github-icon" />
-                  GitHub account | @{user ? user.nickname : ''}
```

Comment:
If the nickname is empty, should we use their user name, full name or something similar as a fallback?